deploy.py

```diff
@@ -43,18 +41,34 @@ def run(build_dir: str) -> None:
         )
     if Path(f"{build_dir}/raw").is_dir():
         # TODO: #7 load_raw only loads one database as configured in ToolGlobals.config, needs more dynamic support
-        load_raw(ToolGlobals, drop=True, file=None, directory=f"f{build_dir}/raw")
+        # Now hardcoded to load into "default" database. This must be configurable in the module (config.yaml)
+        load_raw(
+            ToolGlobals,
+            raw_db="default",
+            drop=drop,
+            file=None,
+            directory=f"f{build_dir}/raw",
+        )
+    # TODO: #21 Implement dry-run consistently across.
     if Path(f"{build_dir}/timeseries").is_dir():
         load_timeseries_metadata(
-            ToolGlobals, drop=True, file=None, directory=f"f{build_dir}/timeseries"
+            ToolGlobals, drop=drop, file=None, directory=f"f{build_dir}/timeseries"
         )
     if Path(f"{build_dir}/transformations").is_dir():
         load_transformations_dump(
-            ToolGlobals, file=None, drop=True, directory=f"{build_dir}/transformations"
+            ToolGlobals, file=None, drop=drop, directory=f"{build_dir}/transformations"
+        )
+    if (models_dir := Path(f"{build_dir}/source_models")).is_dir():
+        load_datamodel_dump(
+            ToolGlobals, drop=drop, directory=models_dir, dry_run=dry_run
+        )
+    if (models_dir := Path(f"{build_dir}/domain_models")).is_dir():
+        load_datamodel_dump(
+            ToolGlobals, drop=drop, directory=models_dir, dry_run=dry_run
         )
-    if (models_dir := Path(f"{build_dir}/data_models")).is_dir():
+    if (models_dir := Path(f"{build_dir}/solution_models")).is_dir():
         load_datamodel_dump(
-            ToolGlobals, drop=True, directory=models_dir, dry_run=True
+            ToolGlobals, drop=drop, directory=models_dir, dry_run=dry_run
         )
```

Comment:
Reintroducing source, domain, and solution models at the top level of a build package?

Reply:
You hadn't merged them into top level, so I just temporary fixed the loading.

Reply:
That wasn't very clearly explained: You HAD merged into top-level, but when doing a build, each of the sub-folders with data models is merged into one for each data model type. We need to implement support for ordering import of modules, but temporary, I just implemented step-wise loading of modules in the deploy.sh script.